axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -184,8 +184,8 @@ def test_all_states_reachable(self):
         owned_states = set(pair[0] for pair in transitions.keys())
 
         un_callable_states = owned_states.difference(called_states)
-        if un_callable_states:
-            raise AssertionError('The following states are un-callable: {}'.format(un_callable_states))
+        error_message = 'The following states are un-reachable: {}'.format(list(un_callable_states))
+        self.assertEqual(error_message, 'The following states are un-reachable: []')
```

Comment:
What's this supposed to be testing? I'm a bit confused by the need for this line...